Zend/Optimizer/zend_func_info.c

```diff
@@ -542,7 +524,6 @@ static const func_info_t func_infos[] = {
 	F1("utf8_decode",                           MAY_BE_STRING),
 
 	/* ext/zlib */
-	F1("gzgetc",                                MAY_BE_FALSE | MAY_BE_STRING),
```

Comment:
Looks like this one got lost -- but it wasn't necessary in the first place. Dropped the group in https://github.com/php/php-src/commit/8c67abb67f7c8c5c5cf9a5214ba81c9be2fae503.